techdocs/docs/getting-started/quick-start-guide.md

```diff
@@ -0,0 +1,32 @@
+# Quick Start Guide
+
+Set up in 2 steps.
+
+### 1. Import the Provided Functions
+
+Add import for the function you need from the [Module Exports](../using-the-package/module-exports.md).
+
+#### `Import Examples:`
+
+```JavaScript
+// ESModule import (preferred).
+import { getIntegration } from '@bcgov/citz-imb-sso-css-api';
+
+// CommonJS import.
+const { getIntegration } = require('@bcgov/citz-imb-sso-css-api');
+```
+
+#### `Use Example:`
+
+```JavaScript
+import { getIntegration } from '@bcgov/citz-imb-sso-css-api';
+
+// Log integration details.
+console.log(await getIntegration());
+```
+
+<br />
+
+### 2. Add Environment Variables
+
+Add the required environment variables from the [Environment Variables](./environment-variables.md) section.
```

Comment:
should we be sending people to another page instead of including the information here on the "Quick Start" page?

Reply:
I like having it separate, especially if users might be wanting to check on the environment variable requirements down the road they just have to find the "Environment Variables" page.